### DISCUSSION TITLE
What do I need to do to get more than 1 member open at a time?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#789] [IBM i] [answered]
I have this problem often. I need to open more than 1 member to copy code from 1 member to another.

I open the first member, then select the second member and open it. Opening the second member closes the first member. 

I need both open to be able to compare the 2 members.

TIA, Darryl Freinkel.

**Comment by worksofliam** [comment, marked as answer]
This is a VS Code setting. Open the settings and search for `workbench.editor.enablePreview`

**Comment by dhfreinkel** [comment]
Found it and try it out.

 

Thank you.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562

**Reply by dhfreinkel** [reply]
Thank you.

**Comment by dhfreinkel** [comment]
Today, something changed. Open tabs are closed when opening a new tab. I tried to find the settings to change but the values do not change.
![Screenshot 2022-08-09 112444](https://user-images.githubusercontent.com/44469913/183690763-6f59fe1c-4766-431e-ba63-04f1032f5571.png)
